iam/policy/admin-action.go

```diff
@@ -85,6 +85,15 @@ const (
 	// GetUserAdminAction - allows GET permission on user info
 	GetUserAdminAction = "admin:GetUser"
 
+	// Cluster Replicate Actions
+
+	// ClusterReplicateAddAction - allow adding replication clusters
+	ClusterReplicateAddAction = "admin:ClusterReplicateAdd"
```

Comment:
It looks like 'Manage' is better than 'Add' here since this action is already used in multiple cases

Reply:
Yeah makes sense @vadmeste

Reply:
We use a single set of service credentials to perform all replication management actions on peers such as creating/removing bucket, making service account updates etc. This service account would likely need only one special permission to perform all these actions - I wasn't sure of a name for this, but I think we can use `ClusterReplicateManage`. Will send an update to the PRs.

Reply:
Do we still need `Add` if we have Manage? @donatello ?

Reply:
Yes, add is to add the cluster repl config, manage is to perform internal replication operations via the svc account created for this purpose. Manage permission just isolates actions by this service account.